packages/engines/src/index.ts

```diff
@@ -68,6 +69,18 @@ path.join(__dirname, '../query-engine-rhel-openssl-1.0.x')
 path.join(__dirname, '../query-engine-rhel-openssl-1.1.x')
 path.join(__dirname, '../query-engine-rhel-openssl-3.0.x')
 
+path.join(__dirname, '../schema-engine-windows.exe')
+path.join(__dirname, '../schema-engine-darwin')
+path.join(__dirname, '../schema-engine-darwin-arm64')
+path.join(__dirname, '../schema-engine-debian-openssl-1.0.x')
+path.join(__dirname, '../schema-engine-debian-openssl-1.1.x')
+path.join(__dirname, '../schema-engine-debian-openssl-3.0.x')
+path.join(__dirname, '../schema-engine-linux-static-x64')
+path.join(__dirname, '../schema-engine-linux-static-arm64')
+path.join(__dirname, '../schema-engine-rhel-openssl-1.0.x')
+path.join(__dirname, '../schema-engine-rhel-openssl-1.1.x')
+path.join(__dirname, '../schema-engine-rhel-openssl-3.0.x')
+
```

Comment:
I don't think this is needed in the end.

Reply:
Was there a reason these were not included though? The comment above says it's for `pkg`, so I found that counter-intuitive.

Reply:
Might not be needed any more indeed, some tech debt for another day.